.bashrc

```diff
@@ -80,8 +82,38 @@ if [ -x /usr/bin/dircolors ]; then
     alias egrep='egrep --color=auto'
 fi
 
+# Initiate virtualenv
 WORKON_HOME="/home/vagrant/.virtualenvs"
 source /usr/local/bin/virtualenvwrapper.sh > /dev/null 2>&1
 cd /vagrant
 
-# the workon command to be added during provisioning 
+# Ask for git configuration data from user based on input
+chmod 755 /home/vagrant/gitconfig
+export PATH=$PATH:/home/vagrant/
+echo 'export PATH=$PATH:/home/vagrant/' >> ~/.profile
+iterator1=0
```

Comment:
But since this is run like a script what is the added benefit of having it being wrapped up in a function. I can do it but it will just add more lines of code and I need to call the function although a simple top level script will do that job too.